rllib/policy/tests/test_compute_log_likelihoods.py

```diff
@@ -216,9 +222,42 @@ def test_sac_discr(self):
 
         do_test_log_likelihood(sac.SAC, config, prev_a)
 
+    def test_cql_cont(self):
```

Comment:
name?

Reply:
Consistent with all the other names in this test:
"cont": continuous actions
"disc": discrete actions